src/ensembl/src/content/app/help/Help.tsx

```diff
@@ -49,12 +53,23 @@ const Help = () => {
     endpoint: `/api/docs/article?url=${encodeURIComponent(location.pathname)}`,
     skip: isIndexPage
   });
+
+  let breadcrumbs: string[] | undefined = [];
+  if (!isIndexPage && menu && article) {
```

Comment:
Why do we exclude the index page from the search, by the way? Won't `buildBreadcrumbs` find the "Overview" item? I would have thought that it should.

Reply:
That's right.  I've updated it.